File: 6KYU/Convert string to camel case/solution.cs
using System;
using System.Linq;
public class Kata
{
	public static string ToCamelCase(string str)
	{
		var split = str.Split(new char[]{'_','-'});
		return split[0] + string.Concat(split.Skip(1).Select(s => s.Substring(0, 1).ToUpper() + s.Substring(1)));
	}
}

File: 6KYU/Convert string to camel case/solution.go
package kata

import "strings"

func ToCamelCase(s string) string {
	splitStr := strings.FieldsFunc(s, func(c rune) bool {
		return c == '-' || c == '_'
	})

	for i, s := range splitStr {
		if i > 0 {
			splitStr[i] = strings.ToUpper(string(s[0])) + s[1:]
		}
	}

	return strings.Join(splitStr, "")
}
